src/us/kbase/auth2/providers/GoogleIdentityProviderFactory.java

```diff
@@ -53,29 +53,35 @@ public IdentityProvider configure(final IdentityProviderConfig cfg) {
 	 */
 	public static class GoogleIdentityProvider implements IdentityProvider {
 	
+		/* might want to take another crack at using the google client. Last time
+		 * trying to figure out how to get it to work was such a pain I gave up and wrote
+		 * my own code in less time than I spent struggling with the client.
+		 */
+		
 		/* Get creds: https://console.developers.google.com/apis
 		 * Google+ API must be enabled
```

Comment:
This is obsolete now, right?

Reply:
Should say Google people api must be enabled. I'll fix it in a later pr.